core/chains/evm/client/errors.go

```diff
@@ -108,7 +108,36 @@ var avalanche = ClientErrors{
 	NonceTooLow: regexp.MustCompile(`(: |^)nonce too low: address 0x[0-9a-fA-F]{40} current nonce \([\d]+\) > tx nonce \([\d]+\)$`),
 }
 
-var clients = []ClientErrors{parity, geth, arbitrum, optimism, substrate, avalanche}
+// Nethermind
+// All errors: https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.TxPool/AcceptTxResult.cs
+// All filters: https://github.com/NethermindEth/nethermind/tree/9b68ec048c65f4b44fb863164c0dec3f7780d820/src/Nethermind/Nethermind.TxPool/Filters
+var nethermindFatal = regexp.MustCompile(`(: |^)(SenderIsContract|Invalid|Int256Overflow|FailedToResolveSender)`)
+
+// Int256Overflow: Calculation of gas price * gas limit + value overflowed int256.
+// though it feels like Int256Overflow could be use for high fees as well?
+var nethermind = ClientErrors{
+	// Assuming all nonce errors are due to the nonce being too low
+	// NonceGap: The nonce is not the next nonce after the last nonce of this sender present in TxPool.
```

Comment:
NonceGap is definitely _not_ a NonceTooLow error, if anything it's the opposite.

Other clients don't give this error... it should probably just not be handled and bucketed as a "temporary" error. If we see it in prod we may have to add some special handling.

Reply:
I've added as a `fatal` error for now, since there's not much context yet here.

Reply:
I dont think it should be fatal. Just leave it unchecked

Reply:
Understood!